### ISSUE TITLE
Tailwind not workin

### ISSUE THREAD

**Issue by sysmat** [shellscape/jsx-email#67]
- Component or Package Name: @jsx-email/tailwind
- Component or Package Version: ^3.0.0
- `@jsxp-email/cli` Version?: ^2.0.0
- Operating System (or Browser): Win 11, Chrome Version 119.0.6045.106
- Node Version: v18.18.2
- Link to reproduction _(⚠️ read below)_:

### Expected Behavior

- to render taillwind button in browser

### Actual Behavior
- not rendered in browser

### Additional Information

- using documentation from https://jsx.email/docs/components/tailwind

```typescript
import { Button } from '@jsx-email/button';
import { Tailwind } from '@jsx-email/tailwind';

const Email = () => {
  return (
    <Tailwind
      config={{
        theme: {
          extend: {
            colors: {
              brand: '#007291'
            }
          }
        }
      }}
    >
      <Button
        href="https://example.com"
        className="bg-brand px-3 py-2 font-medium leading-4 text-white"
      >
        Click me
      </Button>
    </Tailwind>
  );
};
```
- problem is with `import { Tailwind } from '@jsx-email/tailwind'; ` 
- solution importing it from  '@jsx-email/all'
- example in doc is also wrong `const Email` must be `export const Template`

**Comment by shellscape** [comment]
Id really like to help you, but _please_ read instructions. This is what our template says:


> 🚨 Issues WITHOUT a valid reproduction WILL BE CLOSED!
> 
>   Please provide one by:
>   1. Using the StackBlitz reproduction template at https://stackblitz.com/fork/jsx-email-repro
>   2. Provide a minimal repository link (Read https://gist.github.com/Rich-Harris/88c5fc2ac6dc941b22e7996af05d70ff for instructions).
>      Please use NPM for installing dependencies!
>      These may take more time to triage than the other options.
> 
>   ⚠️ ZIP Files are unsafe and maintainers will NOT download them

.

**Comment by shellscape** [comment]
I'm going to unpublish @jsx-tailwind@3.0.0. The `twin` project is an absolute mess, and there were some blind spots that I didn't see putting that together (even though it passed all of our tests). Please use v2.4.4 for now.

**Comment by shellscape** [comment]
I merged this fix and v3.0.1 will be published shortly. If you run into any more problems, downgrade to 2.4.4. It's slower, but will probably resolve any issues you run into.
